src/lib/Components/Bidding/Screens/BillingAddress.tsx

```diff
@@ -85,55 +110,113 @@ export class BillingAddress extends React.Component<BillingAddressProps, Billing
     this.props.navigator.pop()
   }
 
+  selectNextInput = nextComponent => {
+    const inputs = [
+      this.fullName,
+      this.addressLine1,
+      this.addressLine2,
+      this.city,
+      this.stateProvinceRegion,
+      this.postalCode,
+    ]
+
+    for (const input of inputs) {
```

Comment:
In @orta 's Metadata component, it selects the next non-filled input to go to next. We could do that here but for now we just advance through them. I found the other behavior to be a little finicky so I'll keep it simple for now.